app/namespace.go

```diff
@@ -500,6 +520,20 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 					}
 				}
 
+				codecEndpoint := ctx.String(codecEndpointFlagName)
+				// codec server spec is optional, if specified, we need to create the spec and pass along to the API
+				if codecEndpoint != "" {
+					err = validateCodecInput(codecEndpoint)
+					if err != nil {
+						return err
+					}
+					n.Spec.CodecSpec = &namespace.CodecServerPropertySpec{
+						Endpoint:           codecEndpoint,
+						PassAccessToken:    ctx.Bool(codecPassAccessTokenFlagName),
```

Comment:
if someone sets one of these two flags w/o setting the endpoint, should we fail?

Reply:
added error out in this case